server/src/test/java/com/cloud/vm/UserVmManagerImplTest.java

```diff
@@ -1221,4 +1308,174 @@ public void testSetVmRequiredFieldsForImportFromLastHost() {
         Mockito.verify(userVmVoMock).setLastHostId(2L);
         Mockito.verify(userVmVoMock).setState(VirtualMachine.State.Running);
     }
+
+/*
```

Comment:
same here, if this test is not required can you please delete it or remove the comment if we need this.

Reply:
Fixed